examples/testapp/next.config.js

```diff
@@ -1,18 +1,4 @@
 /**
  * @type {import('next').NextConfig}
  */
-module.exports = {
-  async headers() {
-    return [
-      {
-        source: "/",
-        headers: [
-          {
-            key: "Cross-Origin-Opener-Policy",
-            value: "unsafe-none", // Or 'same-origin' to test COOP errors
```

Comment:
Could this still be a valuable example? Maybe we move it instead of removing it?

Reply:
Yeah I think its valuable for testing. Let me add a new PR that adds a headers file that users can import when they want to test coop errs

Reply:
added here [6a2c8b8](https://github.com/coinbase/coinbase-wallet-sdk/pull/1418/commits/6a2c8b83c2e2d5e77a39b2c14be249713465b0eb)